ql/src/java/org/apache/hadoop/hive/ql/exec/MapOperator.java

```diff
@@ -329,6 +330,58 @@ else if (partRawRowObjectInspector.equals(tblRawRowObjectInspector)) {
     return tableDescOI;
   }
 
+  /**
+   * For each source table, combine the nested column pruning information from all its
+   * table scan descriptors and set it in a configuration copy. This is necessary since
+   * the configuration property "READ_NESTED_COLUMN_PATH_CONF_STR" is set on a per-table
+   * basis, so we can't just use a single configuration for all the tables.
+   */
+  private Map<String, Configuration> cloneConfsForNestedColPruning(Configuration hconf) {
```

Comment:
this is what is used before (I think it indicates hadoop configuration) so I didn't change it.